test/puppetlabs/puppetdb/scf/storage_test.clj

```diff
@@ -1829,12 +1829,13 @@
       (let [assert-index-exists (fn [index indexes]
                                   (is (true? (some #(str/includes? % index) indexes))))
 
-            partition (get-partition-names "reports")]
-        ;; check that idx_reports_id index is present in on demand paritions
-        (is (= 1 (count partition)))
-        (dorun (->> partition
-                    (map tu/table-indexes)
-                    (map (partial assert-index-exists "idx_reports_id_")))))))
+            partitions (get-partition-names "reports")]
+        ;; check that primary key index is present in on demand paritions
+        ;; XXX Do we still care about this?
+        (is (= 1 (count partitions)))
+        (doseq [partition-name partitions
```

Comment:
Not sure if this test is still important since we aren't managing the children indices now.

Reply:
Postgres is managing the partition indices automatically, but since this is tied to a former escalation, https://tickets.puppetlabs.com/browse/PDB-4832, I'm going to be conservative and leave the test in place.